images/scripts/debian.setup

```diff
@@ -318,13 +318,6 @@ systemctl disable apt-daily.timer apt-daily-upgrade.timer fstrim.timer man-db.ti
 echo "kernel.core_pattern=|/lib/systemd/systemd-coredump %P %u %g %s %t %c %e" > /etc/sysctl.d/50-coredump.conf
 printf 'DefaultLimitCORE=infinity\n' >> /etc/systemd/system.conf
 
-# disable udev network names, our tests expect the kernel schema
```

Comment:
:clap: I'm glad this gets dropped at last. I'm happy to help with any test fallout.

Please add a detailed commit message for this, in the spirit of "our tests don't need that any more" and "let's test the *actual* OS".